examples/scripts/sentiment_tuning.py

```diff
@@ -178,11 +154,12 @@ def collator(data):
 if ppo_trainer.accelerator.num_processes == 1:
     device = 0 if torch.cuda.is_available() else "cpu"  # to avoid a `pipeline` bug
 ds_plugin = ppo_trainer.accelerator.state.deepspeed_plugin
+task, model_name = args.ppo_config.reward_model.split(":")
 if ds_plugin is not None and ds_plugin.is_zero3_init_enabled():
     with ds_plugin.zero3_init_context_manager(enable=False):
-        sentiment_pipe = pipeline("sentiment-analysis", model=script_args.reward_model_name, device=device)
+        sentiment_pipe = pipeline(task, model=model_name, device=device)
 else:
-    sentiment_pipe = pipeline("sentiment-analysis", model=script_args.reward_model_name, device=device)
+    sentiment_pipe = pipeline(task, model=model_name, device=device)
 
```

Comment:
Here we fixed the merge conflict and made a generalization for @lewtun's new `reward_model_name` arg.